.golangci.yml

```diff
@@ -15,34 +15,25 @@ run:
     - "pkg/apis/eventsources/v1alpha1/zz_generated.deepcopy.go"
 linters:
   enable:
-    - bodyclose
```

Comment:
many linters were failing - so I removed them because there were too many issues to fix - I did fix a couple that looked like code bugs